vite.config.js

```diff
@@ -28,14 +20,20 @@ module.exports = defineConfig({
       },
       external: (id) => {
         if (
-          excluded.includes(id.split('/')[0]) ||
-          (id.startsWith('@') &&
-            excluded.includes(id.split('/').slice(0, 2).join('/')))
+          // direct node_modules import
+          id.includes('node_modules') ||
+          // everything else is library import
+          !(
+            // relative import is ok
+            id.startsWith('.') ||
+            // absolute import is ok
+            id.startsWith('/')
```

Comment:
absolute import is something that should never happen, actually

Reply:
(but maybe paths are transformed to absolute, I don't remember)

Reply:
Some ids are actually absolute paths for some reason, here is data sample

```
styled-components                                                                               
./PopoverRoot                                                                                   
styled-components                                                                               
../button                                                                                       
./PaginationItemStyles                                                                          
./types                                                                                         
./PopoverRoot                                                                                   
../button                                                                                       
/Users/kozlovzxc/Projects/ui/packages/pagination/PaginationItemStyles.tsx                       
/Users/kozlovzxc/Projects/ui/packages/pagination/types.ts                                       
/Users/kozlovzxc/Projects/ui/packages/popover/PopoverRoot.tsx                                   
/Users/kozlovzxc/Projects/ui/packages/button/index.ts                                           
```